### ISSUE TITLE
Kontextabhängige Navigationsleiste

### ISSUE THREAD

**Issue by Glasgesicht** [web-a3/Web-Aufgabe-2#9]
Je nach Seite, auf der sich der Nutzer gerade befindet, sollten die Inhalte der Navigationsleiste angeasst sein. (Da der Nutzer derzeit, z.B. Nicht mehr auf die Starseite zurückkehren kann)

**Comment by Her1726194** [comment]
Ja, die Idee finde ich sehr gut, gerade da ein Zurrückkehren zur Startseite nur über das Logo möglich ist zur Zeit. Setze mich die Tage mal dran.
